CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v6.3.1](https://github.com/voxpupuli/puppet-systemd/tree/v6.3.1) (2024-02-24)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-systemd/compare/v6.3.0...v6.3.1)
+
+**Implemented enhancements:**
+
+- Support `StartLimitIntervalSec` and  `StartLimitBurst` [\#412](https://github.com/voxpupuli/puppet-systemd/pull/412) ([traylenator](https://github.com/traylenator))
+- unit type: add `ConditionPathIsMountPoint` [\#408](https://github.com/voxpupuli/puppet-systemd/pull/408) ([fragfutter](https://github.com/fragfutter))
+- Allow percent \(%\) character in unit names. [\#401](https://github.com/voxpupuli/puppet-systemd/pull/401) ([traylenator](https://github.com/traylenator))
+
+**Fixed bugs:**
+
+- manage\_unit service\_entry Environment can't have multiple values [\#404](https://github.com/voxpupuli/puppet-systemd/issues/404)
```

Comment:
I added the skip-changelog label to this issue because we have the related PR already in the changelog